client/components/telecom/alias/hunting/sounds/telecom-telephony-alias-hunting-sounds.html

```diff
@@ -13,55 +13,55 @@
     </div>
 
     <div class="modal-body clearfix">
-
         <h3 class="modal-title"
             data-translate="telephony_alias_hunting_sounds_manager">
         </h3>
 
-        <input-file data-ng-model="$ctrl.toUpload"
-                    data-ng-accept="audio/*"
-                    data-ng-accept-filter="$ctrl.checkValidAudioExtention(file)">
-            <i class="fa fa-upload mr-1"></i>
-            <span data-translate="telephony_alias_hunting_sounds_upload"></span>
-        </input-file>
+        <div class="form-group mb-5">
+            <input-file data-ng-model="$ctrl.toUpload"
+                        data-ng-accept="audio/*"
+                        data-ng-accept-filter="$ctrl.checkValidAudioExtention(file)">
+                <i class="fa fa-upload mr-1"></i>
+                <span data-translate="telephony_alias_hunting_sounds_upload"></span>
+            </input-file>
 
-        <button type="submit"
-                class="btn btn-default mt-2 float-right no-transition"
-                data-ng-disabled="$ctrl.isUploading"
-                data-ng-if="$ctrl.toUpload">
-            <oui-spinner class="mr-2"
-                         data-ng-if="$ctrl.isUploading"
-                         data-size="s">
-            </oui-spinner>
-            <span data-translate="submit"></span>
-        </button>
+            <button type="submit"
+                    class="oui-button oui-button_secondary mt-2 float-right no-transition"
+                    data-ng-disabled="$ctrl.isUploading"
+                    data-ng-if="$ctrl.toUpload">
+                <oui-spinner class="mr-2"
+                             data-ng-if="$ctrl.isUploading"
+                             data-size="s">
+                </oui-spinner>
+                <span data-translate="submit"></span>
+            </button>
+        </div>
 
-        <div data-ng-if="$ctrl.sounds.length"
-             class="row">
-            <p class="col-xs-12 mt-3"
-               data-translate="telephony_alias_hunting_sounds_list"></p>
+        <div class="oui-list mt-3"
```

Comment:
Instead of applying `oui-list*` classes mixed with Bootstrap rows, I suggest something like:

```diff
-        <div class="oui-list mt-3"
+        <div class="form-group"
              data-ng-if="$ctrl.sounds.length">
-            <p data-translate="telephony_alias_hunting_sounds_list"></p>
-            <ul class="oui-list__items">
-                <li class="oui-list__item row ml-2"
-                    data-ng-repeat="sound in $ctrl.sounds track by $index">
-                    <button type="button"
-                            class="oui-button oui-button_secondary col-sm-10 col-xs-5"
-                            data-ng-click="$ctrl.chooseSound(sound)">
-                        <span data-ng-bind=":: sound.name"></span>
-                    </button>
-                    <button type="button"
-                            class="oui-button oui-button_secondary col-sm-1 col-xs-1"
-                            data-title="{{:: 'telephony_alias_hunting_sounds_delete' | translate }}"
-                            data-ng-disabled="sound.isDeleting"
-                            data-ng-click="$ctrl.deleteSound(sound)">
-                        <oui-spinner data-ng-if="sound.isDeleting"
-                                     data-size="s">
-                        </oui-spinner>
-                        <span class="ovh-font ovh-font-wrong"
-                              data-ng-if="!sound.isDeleting"></span>
-                    </button>
-                </li>
-            </ul>
+            <label class="control-label"
+                   data-translate="telephony_alias_hunting_sounds_list">
+            </label>
+            <div class="btn-group d-block clearfix mb-4"
+                 role="group"
+                 data-ng-repeat="sound in $ctrl.sounds track by $index">
+                <button type="button"
+                        class="btn btn-default w-50"
+                        data-ng-click="$ctrl.chooseSound(sound)"
+                        data-ng-bind=":: sound.name">
+                </button>
+                <button type="button"
+                        class="btn btn-default"
+                        data-title="{{:: 'telephony_alias_hunting_sounds_delete' | translate }}"
+                        data-ng-disabled="sound.isDeleting"
+                        data-ng-click="$ctrl.deleteSound(sound)">
+                    <oui-spinner data-ng-if="sound.isDeleting"
+                                 data-size="s">
+                    </oui-spinner>
+                    <span class="ovh-font ovh-font-wrong"
+                          data-ng-if="!sound.isDeleting">
+                    </span>
+                </button>
+            </div>
         </div>
``` 

Does it fit your needs?

Reply:
It fit my needs :)